### DISCUSSION TITLE
Selecting month with day picker

### DISCUSSION THREAD

**Discussion by Max0u** [gpbl/react-day-picker#1848] [Ideas]
Hello, is there a way to use the date picker to pick month? Something like:
![Capture d’écran du 2023-07-23 22-32-46](https://github.com/gpbl/react-day-picker/assets/21695361/4fcab022-f28d-4977-b728-2848f60a948d)

**Comment by gpbl** [comment]
@Max0u no this is not implemented yet but under consideration...

**Reply by Max0u** [reply]
Awesome! Thanks for the response
